packages/atlas/src/api/queries/fragments.graphql

```diff
@@ -116,7 +116,11 @@ fragment BasicVideoFields on Video {
     ...StorageDataObjectFields
   }
   nft {
-    id # just to check whether NFT exists
+    id
+    creatorRoyalty
+    creatorChannel {
+      title
+    }
```

Comment:
I'm not sure if we should fetch more information in the `BasicVideoFields` fragment. Maybe we could create another query or maybe even use `FullVideoFields` in the NFT Form? @kdembler what do you think?

Reply:
Also, we don't need creatorChannel here, we could use just `BasicChannelFields`

Reply:
As Bartosz suggested, we should be able to use the `video.channel` instead of `video.nft.creatorChannel`. Then, I think it's fine to add only `creatorRoyalty`